package.json

```diff
@@ -49,7 +49,7 @@
     "test:coverage:html": "cross-env NODE_ENV=test BABEL_ENV=coverage nyc mocha packages/material-ui/test/**/*.test.js packages/material-ui/src/{,**/}*.test.js && nyc report --reporter=html",
     "test:karma": "cross-env NODE_ENV=test karma start test/karma.conf.js --single-run",
     "test:regressions": "webpack --config test/regressions/webpack.config.js && rimraf test/regressions/screenshots/chrome/* && vrtest run --config test/vrtest.config.js --record",
-    "typescript": "tsc -p tsconfig.json",
+    "typescript": "tslint -p tsconfig.json \"packages/*/{src,test}/**/*.{ts,tsx}\"",
```

Comment:
Is it intentional that you're not actually using `dtslint` yet?

Reply:
Kind of by accident if that makes sense. This just means that we are only using the rules from `dtslint`.

`dtslint` requires some extra setup because it can't run on more than one package on its own. I was wondering why this worked so seamless all of the sudden.

I guess this is good for now since we do not use `$ExpectError` and target multiple typescript versions which does require `dtslint`.

Reply:
I added it as an explicit dependency for now to avoid further confusion.